client/view/BrowserStats/BrowsersStat.css

```diff
@@ -171,13 +173,13 @@
   display: flex;
   flex-wrap: wrap;
   column-gap: 12px;
-  color: var(--text-secondary);
-  margin-top: auto;
   padding-top: 24px;
+  margin-top: auto;
```

Comment:
Why is there a margin-top?...

Reply:
It is just Stylelint properties sorting.